Merge problem-definition-40 into develop (#45)
SW709E19/Report: report/PartProblemAnalysis/ChapterProblemDefinition/ProblemDefinition.tex

\chapter{Problem Definition}
This chapter presents a definition of the problem derived from the results of the problem analysis.
The essence of the problem is encapsulated in a problem statement defining what this project develops a solution for.

The analysis in \autoref{maintenance} describes the possible economical savings of using a predictive maintenance approach in manufacturing companies.
This is caused by several benefits such as reduction in maintenance costs, increase in production and reduction of downtime.

> However, setting up the currently available solutions, as those described in \autoref{existingsolutions}, can be a time-consuming and non-trivial matter, making the decision of moving to a predictive approach less desireable. % Refactor this! What is the right argument? The source is ONLY neurospace; is that enough? Refer to previous section or provide source?

Reviewer: ```suggestion
```

> In addition to this, the savings a company may experience can only be measured after implementing the strategy, and investing in a full-scale predictive maintenance system is a costly and binding procedure, as \autoref{existingsolutions} described.

Reviewer: ```suggestion
However, the savings a company may experience can only be measured after implementing the strategy, and investing in a full-scale predictive maintenance system is a costly and binding procedure, as \autoref{existingsolutions} described.
```
